src/NHSD.BuyingCatalogue.Solutions.API/ViewModels/Suppliers/GetSupplierResult.cs

```diff
@@ -6,15 +6,15 @@ namespace NHSD.BuyingCatalogue.Solutions.API.ViewModels.Suppliers
     public sealed class GetSupplierResult
     {
         [JsonProperty("description")]
-        public string Description { get; set; }
+        public string SupplierDescription { get; set; }
```

Comment:
Change this back to desc & link, dont need to explicitly say as you are in the supplier